utils/wrappers/packing.go

```diff
@@ -227,6 +227,11 @@ func (p *Packer) PackBytes(bytes []byte) {
 	p.PackFixedBytes(bytes)
 }
 
+// PackBytes append a byte slice to the byte array
+func (p *Packer) PackBytesLength(bytes []byte) {
+	p.PackInt(uint32(len(bytes)))
+}
+
```

Comment:
Does this make sense as a helper function? `I feel like they can just use `PackUint32`.

Reply:
missed that!